src/main/webapp/WEB-INF/tags/instructor/home/coursePanel.tag

```diff
@@ -15,9 +16,30 @@
             <div class="mobile-margin-top-10px col-sm-6">
                 <span class="mobile-no-pull pull-right">
                     <c:forEach items="${courseTable.buttons}" var="button">
-                        <a data-toggle="tooltip" data-placement="top" ${button.attributesToString}>
-                            ${button.content}
-                        </a>
+                        <c:choose>
+                            <c:when test="${fn:length(button.nestedElements) gt 0}">
+                                <div class="dropdown courses-table-dropdown">
+                                    <button class="btn btn-primary btn-xs dropdown-toggle" type="button" data-toggle="dropdown">
```

Comment:
In fact, separator is rendered there because there are two buttons: one is represented via "a" tag (Remind), and other via "button" tag (arrow).
It makes sense for "Remind" button, because click on "Remind"-part has a separate action, while click on arrow opens dropdown list. What about course panel, I don't see why it's needed. What action do you expect e.g. while clicking on "Course"-part ?
Please, clarify this point for me, because currently it seems not reasonable for me